### PR TITLE
Updated the description of max_delta parameter

### PR DESCRIPTION
**Pull request by sushreebarsa**
The documentation currently states that the max_delta parameter controls the maximum absolute change in brightness of images. However, the max_delta parameter actually controls the maximum relative change in brightness of images. This means that the actual change in brightness will depend on the range of values in the input image. So, I have updated the description for the max_delta parameter. Could you please have a look and do the needful? 
Thank you!

### CODE REVIEW

**Comment by cantonios** [comment]
I don't see "absolute" anywhere in the documentation.  Instead, the docs refer to the `adjust_brightness` documentation, which would probably be the place to change if anything.

**Comment by gbaned** [comment]
Hi @sushreebarsa Can you please check @cantonios's [comments](https://github.com/tensorflow/tensorflow/pull/62402#issuecomment-1813075282) and keep us posted ? Thank you!

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -2027,10 +2027,11 @@ def random_brightness(image, max_delta, seed=None):
   with `tf.image.random_*` ops, `tf.image.stateless_random_*` ops guarantee the
   same results given the same seed independent of how many times the function is
   called, and independent of global seed settings (e.g. tf.random.set_seed).
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py:2030]: There should only be one empty line before `Args:`
```suggestion
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -2030,7 +2030,7 @@ def random_brightness(image, max_delta, seed=None):
 
   Args:
     image: An image or images to adjust.
-    max_delta: float, must be non-negative.
+    max_delta: float, must be non-negative. The max_delta parameter controls the maximum relative change in brightness. This means that the actual change in brightness will depend on the range of values in the input image.
```
[the remark this answers is not in the document]
Comment (sushreebarsa) [reply tensorflow/python/ops/image_ops_impl.py]: @mihaimaruseac Please have a look at these changes. Thank you!

[the remark this answers is not in the document]
Comment (mihaimaruseac) [reply tensorflow/python/ops/image_ops_impl.py]:
```
tensorflow/python/ops/image_ops_impl.py:2033:0: C0301: Line too long (110/80) (line-too-long)
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -2030,7 +2030,7 @@ def random_brightness(image, max_delta, seed=None):
 
   Args:
     image: An image or images to adjust.
-    max_delta: float, must be non-negative.
+    max_delta: float, must be non-negative. This parameter controls the maximum relative change in brightness.
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/image_ops_impl.py:2033]:
```suggestion
    max_delta: float, must be non-negative. This parameter controls the maximum
      relative change in brightness.
```

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**